src/psyclone/transformations.py

```diff
@@ -174,21 +171,10 @@ def check_intergrid(node):
 
 
 class LoopFuseTrans(Transformation):
-    ''' Provides a loop-fuse transformation.
-        For example:
-
-        >>> from psyclone.parse.algorithm import parse
-        >>> from psyclone.psyGen import PSyFactory
-        >>> ast,invokeInfo=parse("dynamo.F90")
-        >>> psy=PSyFactory("dynamo0.1").create(invokeInfo)
-        >>> schedule=psy.invokes.get('invoke_v3_kernel_type').schedule
-        >>> schedule.view()
-        >>>
-        >>> from psyclone.transformations import LoopFuseTrans
-        >>> trans=LoopFuseTrans()
-        >>> new_schedule,memento=trans.apply(schedule.children[0],
-                                             schedule.children[1])
-        >>> new_schedule.view()
+    ''' Provides a generic loop-fuse transformation to two Nodes in the
+    PSyIR of a Schedule after performing validity checks for the supplied
+    Nodes. Examples are given in descriptions of children classes
```

Comment:
Change to "in the descriptions of any children classes." and remove the reforence to  the children classes as parent classes should not know what they children classes are.

Reply:
Done.